lib/LedgerSMB.pm

```diff
@@ -495,7 +495,7 @@ sub merge {
         }
         elsif ( !defined $dst_arg && !defined $src->{$arg} )
         {
-            $logger->trace("LedgerSMB.pm: merge setting \$dst_arg is undefined \$src->{\$arg} is undefined");
+            $logger->trace('LedgerSMB.pm: merge setting \$dst_arg is undefined \$src->{\$arg} is undefined');
```

Comment:
Now that interpolation doesn't take place anymore, shouldn't the backslashes be removed?

Reply:
Yes.  Funny, I remember seeing this line and pausing and moving on.  I'm fairly confident that it is the only line with escaped varnames I touched in all this de-interpolation.

Correction pushed.